README.md

```diff
@@ -31,6 +31,12 @@ Finally, run the automated tests with `doit tests`.
 
 For manual testing, you can start the images with `doit up`, however please refer to the next section for a production-ready local deployment of AiiDAlab with aiidalab-launch.
 
+### Test on different architectures
+
+Images are built for multiple architectures (amd64 and aarch64) during continuous integration and pushed to the GitHub Container Registry (ghcr.io).
+To run automated tests or manual test against those images, simply specify the registry with the aformentioned `up` and `tests` commands (e.g. `doit up --registry=ghcr.io/).
+Note: You may have to [log into the registry first](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-to-the-container-registry).
```

Comment:
Maybe just put the command here. Took me some time to find it in the link.

```suggestion
Note: You may have to log into the registry first (`docker login ghcr.io -u USERNAME`).
```

Reply:
I'd prefer not to, because the linked page provides more context, e.g., that one should obtain a PAT and with which access rights etc. I think by just showing a single command here – which may or may not work – we are oversimplifying this.

Reply:
ok, sounds good to me.